### ISSUE TITLE
Does webview work?

### ISSUE THREAD

**Issue by haswalt** [ardera/flutter-pi#405]
Does a simople webview work? I've tried a couple of different implementations incuding flutters own and a linux version built on CEF.

**Comment by thomaswiradikusuma** [comment]
@haswalt Does it work?

**Comment by ardera** [comment]
No, it does not work. The CEF-based webview for linux desktop only works with the linux desktop embedder unfortunately.
